revoke/revoke.go

```diff
@@ -142,6 +158,31 @@ func certIsRevokedCRL(cert *x509.Certificate, url string) (revoked, ok bool) {
 		}
 	}
 
+	u, err := neturl.Parse(url)
+	if err != nil {
+		log.Warningf("failed to parse CRL url: %v", err)
+		return false, false
+	}
+	if u.Scheme == "" && shouldFetchCRL {
```

Comment:
file scheme doesn't work with `ioutil.ReadFile`. And do you really want to convert regular path to `file://` and then convert it back for `ReadFile`?